src/input/return_enums.rs

```diff
@@ -121,16 +123,15 @@ impl<'py> GenericIterable<'_, 'py> {
     }
 }
 
-#[derive(Debug)]
-struct MaxLengthCheck<'a, INPUT> {
+struct MaxLengthCheck<'a, INPUT: ?Sized> {
```

Comment:
I wonder if the lack of `Debug` impl could be annoying down the road.

Is there any way to build with `Debug` applied to everything?

Reply:
Eurgh I mean I could have `INPUT: Debug + ?Sized` and then the derive will work again :)

Reply:
don't worry if you're happy.